pages/index.tsx

```diff
@@ -50,11 +66,46 @@ const Home: NextPage = () => {
         <div className="flex justify-center content-center h-screen translate-y-">
           <div className="h-1/4 w-full relative m-auto">
             <Card className="bg-light relative sm:w-5/12 overflow-visible drop-shadow-lg rounded-xl bg-opacity-50 m-auto xs:w-11/12">
-              <div className="bottom-0 absolute text-dark w-full h-1/4 m-auto pb-12 mt-4 mb-4">
+              <div className="bottom-0 absolute text-dark w-full h-1/4 m-auto pb-12 mt-4 mb-6">
                 <SplashPageSearchBar
                   selectSearchValue={searchOptionChosen}
                   disabled={false}
                 />
+                <TransparentTooltip
+                  title={
+                    <Card className="px-3 py-2" elevation={darkModeElevation}>
```

Comment:
What is the purpose of the "elevation" prop?

Reply:
elevation controls the amount of box shadow around the Card. Larger elevation means larger shadow